local/config/vufind/searchspecs.yaml

```diff
@@ -356,6 +356,8 @@ AllFields:
     - description
     - isbn
     - issn
+    - key_word_chains
+    - key_word_chain_bag
```

Comment:
The topic field contains keyword chains, doesn't it? I don't think we need this one.